web_export_html_as_text/README.rst

```diff
@@ -0,0 +1,95 @@
+=======================
+Web Export Html As Text
+=======================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:2146e2848f91f88561de66315cd667a5da2271a1803285722619230788424ecd
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/16.0/web_export_html_as_text
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_export_html_as_text
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/web&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module allows exporting HTML fields as plain text.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To export HTML fields as plain text:
+
+- Open any list view.
+- Select the records, then click Export under the Action menu.
+- Enable "Export HTML as text" in the export dialog.
+
+Note: This setting will apply to all HTML fields exported in the current export dialog.
+
+Known issues / Roadmap
+======================
+
+Rich formatting (bold, italic, underline, colors, fonts, alignment) cannot be preserved in exports.
```

Comment:
It is not possible to keep a list such as
1.
2.
3.
All of them will be in a single line.
I thought it would be better to add this content as well.

Reply:
Done!